schema/crdb/11.0.0/up01.sql

```diff
@@ -0,0 +1,23 @@
+/*
+ * Drop the entire metric producer assignment table.
+ *
+ * Programs wishing to produce metrics need to register with Nexus. That creates
+ * an assignment of the producer to a collector, which is recorded in this
+ * table. That registration is idempotent, and every _current_ producer will
+ * register when it restarts. For example, `dpd` includes a task that registers
+ * with Nexus, so each time it (re)starts, that registration will happen.
+ *
+ * With that in mind, dropping this table is safe, _because all updates are
+ * currently offline_. The current metric producers are:
```

Comment:
```suggestion
 * With that in mind, dropping this table is safe, because as of today all software updates require that the whole control plane be offline, so we know that these entries will be re-created shortly. The current metric producers are:
```